package.json

```diff
@@ -73,9 +73,13 @@
   "dependencies": {
     "arr-flatten": "^1.0.1",
     "ava-init": "^0.1.0",
-    "babel-core": "^5.8.23",
-    "babel-plugin-espower": "^1.1.0",
-    "babel-runtime": "^5.8.29",
+    "babel-core": "6.0.14",
```

Comment:
@jamestalmage Yeah, was only commenting what I want the final PR to look like. We can wait on doing it until this PR is ready.